### PR TITLE
Api requests

### PR DESCRIPTION
**Pull request by ArturBa**

### CODE REVIEW

File: src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -18,10 +21,17 @@ export class DashboardComponent implements OnInit {
    * User  city name
    */
   city = '';
+  /**
+   * Forecast data
+   */
+  forecast: ForecastModel = new ForecastModel();
```
Comment (ArturBa) [inline src/app/dashboard/dashboard.component.ts:27]: forecast: ForecastModel;
there is no need for default empty value I guess

File: src/app/services/open-weather/URL.ts [hunk 2]
```diff
@@ -11,7 +11,11 @@ export const API_URL = {
     'https://api.openweathermap.org/data/2.5/onecall?lat={lat}&lon={lon}' +
     '&exclude=minutely,daily,alerts&units=metric&appid=' +
     environment.weatherapi.apiKey,
-  air:
+  air_current:
```
Comment (ArturBa) [inline src/app/services/open-weather/URL.ts:14]: air_pollution?

File: src/app/dashboard/dashboard.component.ts [hunk 3]
```diff
@@ -18,10 +21,17 @@ export class DashboardComponent implements OnInit {
...
   isDataReady = false;
 
-  constructor(protected ipApiService: IpApiService) {}
+  constructor(
+    protected ipApiService: IpApiService,
+    protected owService: OpenWeatherService
+  ) {}
 
   async ngOnInit(): Promise<void> {
     this.updateData();
```
Comment (ArturBa) [inline src/app/dashboard/dashboard.component.ts:37]: we should start by initLocation and then set dataReady to true.
Please validate if this works

**Review by ArturBa — COMMENTED** [review]
Please make fallowing changes

File: src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -13,47 +21,119 @@ export class DashboardComponent implements OnInit {
...
+    this.updateForecast();
+  }
+
+  /**
+   * Update forecast for given coords
+   */
+  protected updateForecast(): void {
+    this.setDataLoading();
+
+    this.requestForecastData().subscribe((data) => {
```
Comment (ArturBa) [inline src/app/dashboard/dashboard.component.ts:99]: not sure if there is no nice way to split this into `[weather_resp, air_resp]`

File: src/app/forecast/forecast/forecast.component.spec.ts [hunk 5]
```diff
@@ -18,7 +18,7 @@ describe('ForecastComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  xit('should create', () => {
```
Comment (ArturBa) [inline src/app/forecast/forecast/forecast.component.spec.ts:21]: most of the test would be fixed after merging weather weekly forecast (testHelper)

**Review by BartlomiejWojcik — APPROVED** [review]
Looks good to me

**Pull request merged**